### PR TITLE
Adding Swagger client

### PR DESCRIPTION
**Pull request by dewmini**

### CODE REVIEW

**Comment by bhathiya** [comment]
Shouldn't these go to features directory?

**Comment by chanaka3d** [comment]
Merging the pull request for now since we are maintaining the UUF apps and components inside /components/web directory.

File: components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/js/application.js [hunk 1]
```diff
@@ -1,3 +1,39 @@
...
+                    $.ajax({
+                        url: '/store/public/components/root/base/templates/appListRow.hbs',
+                        type: 'GET',
+                        success: function (result) {
+                            var theTemplateScript = result;
+                            var theTemplate = Handlebars.compile(theTemplateScript);
+                            var theCompiledHtml, context;
+
+                            for (var i in data.obj.list) {
+
```
Comment (prasa7) [inline components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/js/application.js]: Remove unnecessary blank lines

File: components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/templates/appListRow.hbs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    <td>{{status}}</td>
+    <td>{{subscriptions}}</td>
+    <td>
+        <a title="View" href="/store/application/{{name}}"
+           class="btn  btn-sm padding-reduce-on-grid-view">
+      <span class="fw-stack">
+          <i class="fw fw-ring fw-stack-2x"></i>
+          <i class="fw fw-view fw-stack-1x"></i>
+      </span>
+            <span class="hidden-xs">View</span>
```
Comment (prasa7) [inline components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/templates/appListRow.hbs]: Add multi language support for this texts. Check the whole PR and update.

File: components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/js/application.js [hunk 3]
```diff
@@ -1,3 +1,39 @@
+$(window).load(function () {
+
+    var client = new SwaggerClient({
+        url: 'https://apis.wso2.com/api/am/store/v0.10/swagger.json',
```
Comment (prasa7) [inline components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/js/application.js]: Shall we move this SwaggerClient url to a config file. If we do this we will only need to change this location only in one place if we are chaining this URL.

File: components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/templates/appListRow.hbs [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+      </span>
+            <span class="hidden-xs">Edit</span>
+        </a>
+        <a title="Delete" href="#" data-id="DefaultApplication" data-count="1"
+           class="btn btn-sm padding-reduce-on-grid-view deleteApp">
+      <span class="fw-stack">
+          <i class="fw fw-ring fw-stack-2x"></i>
+          <i class="fw fw-delete fw-stack-1x"></i>
+      </span>
+            <span class="hidden-xs">Delete</span>
```
Comment (prasa7) [inline components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/templates/appListRow.hbs]: Add multi language support

File: components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/js/application.js (hunk 3 above)
Comment (dewmini) [inline components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/js/application.js]: This code has been moved to public/js/applications/index.js. Fixed in https://github.com/wso2/carbon-apimgt/commit/82f2e62ad13d082ec487bb416a69a72df180d322

File: components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/js/application.js (hunk 1 above)
Comment (dewmini) [inline components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/js/application.js]: Fixed in https://github.com/wso2/carbon-apimgt/commit/82f2e62ad13d082ec487bb416a69a72df180d322

File: components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/templates/appListRow.hbs (hunk 2 above)
Comment (dewmini) [inline components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/templates/appListRow.hbs]: With new UUF features, files within public folder will not be used. This code will be removed.

File: components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/templates/appListRow.hbs (hunk 4 above)
Comment (dewmini) [inline components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/templates/appListRow.hbs]: With new UUF features, files within public folder will not be used. This code will be removed.

**Pull request merged**